### PR TITLE
Update yarn.lock with AnyEntity caster change and turn yarn audit on for travis

### PR DESCRIPTION
**Pull request by hossenlopp**
Pull requests into cqm-execution require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**BEFOR MERGE**
- [x] After merge of https://github.com/projecttacoma/cqm-models/pull/128 update package.json back and rebuild browser.js.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this PR: https://jira.mitre.org/browse/BONNIE-2059 https://jira.mitre.org/browse/BONNIE-2087
- [x] Internal ticket links to this PR
- [x] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass

**Reviewer 1:**

Name: @losborne 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**Reviewer 2:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@ cache:
     - "node_modules"
 script:
   - yarn install
+  - yarn audit
```
Comment (losborne) [inline .travis.yml]: 🥇 good catch
